product-scenarios/3-securing-an-api-with-different-security-protocols/3.2-api-developers-want-to-secure-their-api-using-an-user-access-control-machanism/3.2.1-api-developer-wants-to-secure-api-resources-using-user-roles/src/test/java/org/wso2/am/scenario/tests/api/secure/userRoles/SecureUsingUserRolesTestCases.java

```diff
@@ -136,13 +182,42 @@ private void deleteRoles() throws APIManagementException {
         }
     }
 
+    private void createAndPublishAPI()throws APIManagerIntegrationTestException{
```

Comment:
Will you be able to move this method to the same util class I've mentioned below. We can create the API request in the test case and pass it to the util method

Reply:
This method removed and used a common method.